rules/windows/discovery_high_number_ad_properties.toml

```diff
@@ -0,0 +1,67 @@
+[metadata]
+creation_date = "2023/01/29"
+integration = ["windows", "system"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/04/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identify read access to a high number of Active Directory object attributes. The knowledge of objects properties can
+help adversaries find vulnerabilities, elevate privileges or collect sensitive information.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.*"]
```

Comment:
If Elastic Defend compatible, add index pattern. Can we tighten the scope on `logs-system.*` as well to `logs-system.security*`?

Reply:
tightened system index